<launch>

  <!-- Specify the out folder name -->
  <arg name="save_dir" default="$(find auto_calibration_tools)/bag_extraction/motion_lab1_small"/>
  <arg name="save_step" default="5"/>


  <!-- Specify the directory where you want to save the bag file -->
  <arg name="bag_dir" default="$(find auto_calibration_tools)/bags"/>
  <arg name="input_bag_file" default="motion_lab1_2.bag" />

      <!-- Declare the node -->
  <node
    name="sync_node"
    pkg="auto_calibration_tools"
    type="offline_time_synch.py"
    output="screen">
    <param name="save_dir" value="$(arg save_dir)"/>
    <param name="save_step" value="$(arg save_step)"/>
  </node>

  <!-- Launch the ROS bag player  -->
  <node name="bag_player" pkg="rosbag" type="play" args="$(arg bag_dir)/$(arg input_bag_file)">
    <param name="pause" value="false" />
  </node>


</launch>
